go/vt/schemadiff/errors.go

```diff
@@ -239,3 +238,11 @@ func (e *InvalidColumnInForeignKeyConstraintError) Error() string {
 	return fmt.Sprintf("invalid column %s referenced by foreign key constraint %s in table %s",
 		sqlescape.EscapeID(e.Column), sqlescape.EscapeID(e.Constraint), sqlescape.EscapeID(e.Table))
 }
+
+type ViewDependencyUnresolvedError struct {
+	View string
```

Comment:
Can we also here have a reference to the entity it tries to refer to? Like if a another table / view is missing, we'd also know the name here as well? 

We could then further improve the error to "view %s references unknown entity %s" or something along those lines? 

And maybe different from having a loop and have a separate error type for that?

Reply:
So the way we detect these dependencies is by creating a connected graph, and then looking at "what's outside the graph". We don't explicitly then check which tables/views are referenced by the left-out views. We could definitely do that.

Reply:
@dbussink let's iterate on that in a followup PR though, if that's OK

Reply:
Yeah, we can improve on a further iteration.